projects/kubernetes/kube-master.yml

```diff
@@ -16,14 +16,14 @@ onboot:
   - name: format
     image: linuxkit/format:84a997e69051a1bf05b7c1926ab785bb07932954
   - name: mounts
-    image: linuxkit/kubernetes:latest-mounts
-    capabilities:
-     - all
-    pid: host
-    rootfsPropagation: shared
+    image: linuxkit/mount:ac8939c4102f97c084d9ddfd445c1908fce6d768
+    command: ["/mount.sh", "/var/lib/"]
+  - name: var
+    image: library/alpine:3.6
+    command: ["mkdir", "/var/lib/kubeadm"]
     binds:
-     - /dev:/dev
-     - /var:/var:rshared,rbind
+     - /var/lib:/var/lib:rshared,rbind
+    rootfsPropagation: shared
```

Comment:
`mkdir` does not need shared root propagation, or `rshared` options, it is not mounting anything